src/Libraries/DSOffice/Excel.cs

```diff
@@ -820,6 +820,30 @@ public static object[][] ImportExcel(FileInfo file, string sheetName, bool readA
             return Excel.ReadFromFile(file, sheetName, readAsStrings, showExcel);
         }
 
+        /// <summary>
+        ///     Write data to a Microsoft Excel spreadsheet. Data is written by row
+        ///     with sublists to be written in successive rows. Rows and columns are
+        ///     zero-indexed; for example, the value in the data list at [0,0] will
+        ///     be written to cell A1. Null values and empty lists are written to Excel 
+        ///     as empty cells. This node requires Microsoft Excel to be installed. 
+        /// </summary>
+        /// <param name="filePath">File path to the Microsoft Excel spreadsheet.</param>
+        /// <param name="sheetName">Name of the workseet to write data to.</param>
+        /// <param name="startRow">Start row for writing data. Enter 0 for Row 1, 1 for Row 2, etc.</param>
+        /// <param name="startCol">
+        ///     Start column for writing data. Enter 0 for Column A, 1 for Column B, etc.
+        /// </param>
+        /// <param name="data">Data to write to the spreadsheet.</param>
+        /// <param name="overWrite"></param>
+        /// <returns name="data">Data written to the spreadsheet.</returns>
+        /// <search>office,excel,spreadsheet</search>
+        [Obsolete("Use ExportToExcel instead.")]
```

Comment:
Does adding the `Obsolete` attribute also hide it from the library or do we also need to add the `[IsVisibleInDynamoLibrary(false)]` attribute?

Reply:
Yes obsolete hides it from the Library.